### PR TITLE
Added code for training the model

### PR DESCRIPTION
**Pull request by tamanna-sb**
I have added a function for training the model and set hyperparameters. Here is the details:

**Hyperparameters:**
1. EMBEDDING_DIM 
2. batch_size
3. epochs 
4. rnn_units 

**Model:**
1. Created a Sequential Model 
2. Added Masking Layer
3. Added Bi-LSTM Layer
4. Added Dense layer and used softmax for activation

**Loss Function:** binary_crossentropy
**Optimizer:** rmsprop

### CODE REVIEW

**Comment by RubyaAfrin** [comment]
Thank you for your contribution. You have made good assumption of Hyperparameters. I will included the file to my source code for training the model.
